cmd/server/http/http.go

```diff
@@ -728,3 +726,26 @@ func release(payload *payload, flowSvc *flow.Service) http.HandlerFunc {
 func convertTimeToEpoch(t time.Time) int64 {
 	return t.UnixNano() / int64(time.Millisecond)
 }
+
+type commitInfo struct {
+	AuthorName  string
+	AuthorEmail string
+	Service     string
+}
+
+func extractAuthorFromCommit(message string) (commitInfo, error) {
+	pattern := `^\[(?P<service>.*)\].*\nSigned-off-by:\s(?P<authorName>.*)\s<(?P<authorEmail>.*)>`
+	r, err := regexp.Compile(pattern)
+	if err != nil {
+		return commitInfo{}, errors.WithMessage(err, "regex didn't match")
```

Comment:
This does in fact not decide if it matches but only verify the regular expression is valid and compilable.

If we compile it ahead of time this check can be removed.